src/utils/addresses.ts

```diff
@@ -73,3 +77,17 @@ export const sortAddressList = (addresses: Address[]): Address[] =>
     if (b.settings.isMain) return 1
     return (b.lastUsed ?? 0) - (a.lastUsed ?? 0)
   })
+
+export const migrateAddressMetadata = () => {
```

Comment:
It's possible there are more migrations in the future; because of this we now need to make descriptive and composable migrations. It's possible a user can be anywhere in the migration process so each migration task must stay for the application's life.

Here's an example (there are most likely things it's forgetting):

```
function migration20220509_MoveToNewStorageKeyName() { ... }
function migration20220811_AddSomePropertyToAddresses() { ... }

function migrate() {
  migration20220509_MoveToNewStorageKeyName()
  migration20220811_AddSomePropertyToAddresses()
}
```

This is standard for any software which has migration processes.

Reply:
> It's possible a user can be anywhere in the migration process so each migration task must stay for the application's life.

Yes, that's totally true!

I don't think it's necessary to introduce these weird function names on the very first migration function. Maybe it's worth it when there is more than one? I'd leave it as it is now.

Reply:
Sure; this'll have to be remembered in the future though by not just you but anyone who may take the issue. :slightly_smiling_face: